ci_jobs_trigger/app.py

```diff
@@ -3,12 +3,21 @@
 from simple_logger.logger import get_logger
 from flask.logging import default_handler
 
+from ci_jobs_trigger.libs.openshift_ci_re_trigger.openshift_ci_re_trigger import (
```

Comment:
if you put `openshift_ci_re_trigger` code under folder please do the same for `zstream_trigger`